app/ota_client.py

```diff
@@ -1292,8 +1320,158 @@ def _rollback(self):
             #
```

Comment:
`self._ota_status.set_ota_status(next_state)`
should be `self.set_ota_status(next_state)`